crate_universe/Cargo.toml

```diff
@@ -14,13 +14,13 @@ edition = "2018"
 # Several of these dependencies need to be compatible with the version cargo-raze is using.
 
 anyhow = "1"
-cargo-raze = "0.9"
-cargo_metadata = "0.12.1"
-cfg-expr = "0.4.1"
-env_logger = "0.7"
+cargo-raze = "0.11"
```

Comment:
Should this be pointing at a `git` ref or similar, rather than a released version?

Reply:
Ah, I thought that was in this PR. My bad, I'll go pull that out of a future commit

Reply:
Updated.